notebooks/community/ml_ops/stage3/get_started_with_automl_pipeline_components.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>The ### Delete ... looks weird. Is there a format error with the heading?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/440/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>

Reply:
There aren't any formatting errors noticed in the notebook : 
<img width="629" alt="stage3_automl_pipelines" src="https://user-images.githubusercontent.com/16080687/161598247-23ae28eb-0bc5-4e8a-bc2b-639760a7c9a5.PNG">
Is it because there are two cells with descriptions under the cleaning up section? @andrewferlitsch , could you please elaborate more on the issue?

Reply:
must have been reviewNB rendering the cell wierd.